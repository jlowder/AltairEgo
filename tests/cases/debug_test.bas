10 FOR I=1 TO 2 : PRINT "I=";I : GOSUB 100 : PRINT "AFTER GOSUB" : NEXT I
20 PRINT "DONE"
30 END
100 PRINT "SUBROUTINE"
110 RETURN
