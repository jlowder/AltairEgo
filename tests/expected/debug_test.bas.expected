Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
I= 1
SUBROUTINE
AFTER GOSUB
I= 2
SUBROUTINE
AFTER GOSUB
DONE
OK
